### ISSUE TITLE
Inline Mode

### ISSUE THREAD

**Issue by francoisk6** [tamble/jquery-ui-daterangepicker#43]
is there anyway to display it as inline mode?

**Comment by op16** [comment]
No. Presumably it's implemented in [some fork](https://github.com/dfmiller/jquery-ui-daterangepicker).

**Comment by Paul424** [comment]
Just wanted to mention that i pushed my edits to support inline mode on my fork: https://github.com/Paul424/jquery-ui-daterangepicker however it's not mergable; i had to do many edits to make it suitable for my application. Still it could help...

Also when it helps i can also provide the django widget i created for it (to parse the range set to the input).
